### PR TITLE
fix build issues by using alpha version

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
The master branch is completely messed up, so we need to use an alpha build to fix it. After this PR is merged we can eliminate the alpha dependency. 

we don't need a semantic release fix commit because it will release based on previous commits already in master once the build succeeds.

### CODE REVIEW

File: packages/otp2-tile-overlay/package.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
     "react-map-gl": "^7.0.15"
   },
   "dependencies": {
-    "@opentripplanner/map-popup": "^4.0.0"
+    "@opentripplanner/map-popup": "^v3.2.0-alpha.1"
```
Comment (daniel-heppner-ibigroup) [inline packages/otp2-tile-overlay/package.json:19]: this is the fix

**Review by amy-corson-ibigroup — APPROVED** [review]
thank you! And you don't need to update snapshots, right?

**Comment by daniel-heppner-ibigroup** [comment]
So, it looks like Storybook was finding the .story files buried inside node modules. This was happening because of the copy all files flag we added to Babel to support the custom image in transitive overlay. I clamped down the glob used to find stories a bit by telling it to only look in the src directories of packages. Is this too restrictive? Let me know maybe we can adjust it. 

That image in the overlay keeps biting us

**Review by miles-grant-ibigroup — APPROVED** [review]
Great fix thanks. That glob wasn't reviewed properly when we approved it! This one looks good to me.

**Pull request merged**